### PR TITLE
AVIATORAPI-93 List tenants with access to private flavor

### PR DESCRIPTION
**Pull request by lvillasica**

### CODE REVIEW

**Comment by jocaba** [comment]
+1

File: lib/aviator/openstack/compute/v2/admin/list_flavor_tenants.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    end
+
+
+    def http_method
+      :get
+    end
+
+
+    def url
+      str  = "#{ base_url }/flavors"
```
Comment (choodur) [inline lib/aviator/openstack/compute/v2/admin/list_flavor_tenants.rb]: Maybe having this is as a single string is better and makes it more readable, what do you think?

Comment (lvillasica) [inline lib/aviator/openstack/compute/v2/admin/list_flavor_tenants.rb]: Yeah I think so. This was due to some UNICODE error I encountered that I forgot to update this part but will do. Thanks for noticing. :ok_hand:

**Comment by lvillasica** [comment]
@musashi-dev/ronins 
updated. please review

**Comment by choodur** [comment]
Looks good to me.

**Pull request merged**